core/flyout_button.js

```diff
@@ -138,20 +138,28 @@ Blockly.FlyoutButton.prototype.createDom = function() {
   if (!this.isLabel_) {
     // Shadow rectangle (light source does not mirror in RTL).
     var shadow = Blockly.utils.createSvgElement('rect',
-        {'class': 'blocklyFlyoutButtonShadow',
-         'rx': 4, 'ry': 4, 'x': 1, 'y': 1},
-         this.svgGroup_);
+        {
+          'class': 'blocklyFlyoutButtonShadow',
+          'rx': 4, 'ry': 4, 'x': 1, 'y': 1
+        },
+        this.svgGroup_);
   }
   // Background rectangle.
   var rect = Blockly.utils.createSvgElement('rect',
-      {'class': this.isLabel_ ?
-       'blocklyFlyoutLabelBackground' : 'blocklyFlyoutButtonBackground',
-       'rx': 4, 'ry': 4},
+      {
+        'class': this.isLabel_ ?
+          'blocklyFlyoutLabelBackground' : 'blocklyFlyoutButtonBackground',
```

Comment:
Another two spaces here?

Reply:
Right now I have ternary expression indentation checking turned off.  I don't remember my reasoning--I think I was just having trouble matching either of the options.  Here's the rule for it: https://eslint.org/docs/rules/indent#flatternaryexpressions.  I have "ConditionalExpression" in my "ignoredNodes" for now. 

I'll add the spaces here, and maybe later I'll turn ternary checking back on.